src/frontend/src/lib/components/ui/ThreeBackground.svelte

```diff
@@ -152,8 +152,18 @@ void main(){
 		camera.aspect = container.clientWidth / container.clientHeight;
 		camera.updateProjectionMatrix();
 	};
+
+	const resizeAction = (node: HTMLElement) => {
+		const resizeObserver = new ResizeObserver(handleResize);
+		resizeObserver.observe(node);
+		return {
+			destroy() {
+				resizeObserver.unobserve(node);
```

Comment:
Shouldn't the observer be disconnected as well?

Reply:
yes! i forgot to put it in this iteration!